README.md

```diff
@@ -635,6 +635,7 @@ nspanel_popup_notification:
   mode: single
   icon: mdi:message-badge
  ```
+Note: No Buzzer may be connected to the panel. See: https://blakadder.com/nspanel-teardown/
```

Comment:
Is there any other source for this information? The buzzer actually IS present even on the disassembled unit, yet it's on the touch panel PCB.

Reply:
My unit also has a buzzer, maybe it was only blakadder's pre production unit